icu4c/source/i18n/plurrule_impl.h

```diff
@@ -30,6 +30,14 @@
 #include "hash.h"
 #include "uassert.h"
 
+#ifndef U_HIDE_INTERNAL_API
```

Comment:
please remove this guard -- unnecessary in an internal header
ok to `git commit --amend`

Reply:
Done.